PocketKit/Sources/PocketKit/Item/Item+Extensions.swift

```diff
@@ -18,12 +18,30 @@ public extension SavedItem {
     var isPending: Bool {
         item == nil
     }
+
+    var shouldOpenInWebView: Bool {
+        item?.shouldOpenInWebView == true
+    }
+
+    var isSyndicated: Bool {
+        item?.isSyndicated == true
+    }
 }
 
 public extension Item {
     var shouldOpenInWebView: Bool {
-        return isArticle == false
-        || hasImage == .isImage
-        || hasVideo == .isVideo
+        if !isSaved && !isSyndicated {
+            return true
+        }
+
+        return !isArticle || hasImage == .isImage || hasVideo == .isVideo
```

Comment:
Can you add some comments explaining the conditionals here? Unfortunately the graphs is not as descrptive as we'd all like for this.

Something like:

If the the item is not an article, or it is an image, or it is a video. (i assume that's what it is doing?)

Reply:
Before I go the comment route, let me see if the code can be written in a more understandable way.

Reply:
Ok, nm. The logic is confusing enough that [comments helped explain it](https://github.com/Pocket/pocket-ios/pull/246/commits/d81374da79c45588655f84ba3bfe8f12914dfba4)

Reply:
thank you!